dapr/aio/clients/grpc/client.py

```diff
@@ -1103,6 +1108,260 @@ async def unlock(
         return UnlockResponse(status=UnlockResponseStatus(response.status),
                               headers=await call.initial_metadata())
 
+    async def start_workflow(
+            self,
+            workflow_component: str,
+            workflow_name: str,
+            input: Optional[bytes],
```

Comment:
`bytes` is not a user-friendly input type. Most users would prefer `Any` and rely on use to do JSON serialization since that's what the workflow authoring SDKs generally expect. I think we can support both by making this `Union[Any, bytes, None] = None`.
```suggestion
            input: Union[Any, bytes, None] = None,
```

Then, down below, only serialize to JSON if the input type is something other than `bytes` - i.e.

```python
if isinstance(input, bytes):
    encoded_data = bytes
else:
    encoded_data = json.dumps(input).encode("utf-8") if input is not None else None
```